packages/@uppy/transloadit/src/index.js

```diff
@@ -251,6 +254,42 @@ module.exports = class Transloadit extends Plugin {
     })
   }
 
+  _createAssemblyWatcher (assemblyID, fileIDs, uploadID) {
+  // AssemblyWatcher tracks completion states of all Assemblies in this upload.
+    const watcher = new AssemblyWatcher(this.uppy, assemblyID)
+
+    watcher.on('assembly-upload', (id) => {
```

Comment:
When I try it with this PR and `waitForEncoding: true`, the postprocessing step is never entered. I think it's because the `assembly-upload` event is not actually fired anywhere right now.

Reply:
I think the most _accurate_ moment to fire that event would be when all the files belonging to an assembly have finished uploading. An easier way would be to fire it when `uppy.on('transloadit:assembly-executing')` fires, which is when Transloadit tells us that it has received all the files. That's typically going to be a few ms later but it might be OK.